connectionRequester/connectionRequester.go

```diff
@@ -0,0 +1,191 @@
+package connectionRequester
```

Comment:
Good Go package names (and filenames) are typically all lower case. `connectionrequester` or `connrequester` would both be fine.

https://blog.golang.org/package-names